purchases/src/main/kotlin/com/revenuecat/purchases/IdentityManager.kt

```diff
@@ -20,6 +20,7 @@ internal class IdentityManager(
             ?: generateRandomID()
         debugLog("Identifying App User ID: $appUserIDToUse")
         deviceCache.cacheAppUserID(appUserIDToUse)
+        deviceCache.migrateSubscriberAttributesIfNeeded(appUserIDToUse)
```

Comment:
I think we might need to pass here the cached ID instead 🤔 Otherwise the previous attributes get associated to this new appUserID, but they might be from the previous user id? maybe I am thinking too much

Reply:
`migrateSubscriberAttributesIfNeeded` would always move subscriber attributes from the same user, so if `cachedUserId != appUserIDToUse`, the ones for `cachedUserId` would be untouched, at least on the iOS implementation (I still need to review this one)

Reply:
is there an edge case I'm missing?